### DISCUSSION TITLE
Poll and options (anonymous)

### DISCUSSION THREAD

**Discussion by eutervogel** [windkh/node-red-contrib-telegrambot#346] [Q&A] [answered]
If I understand this(https://flows.nodered.org/node/node-red-contrib-telegrambot) correctly, the method: poll should receive an object with question and options as answer choices in msg.payload.content, and msg.payload.options should include additional options such as is_anonymous: false.

Like so:
```
msg.payload = {
     content: {
          question: "What's up bro?".
          options: ["Not much.", "A lot."]
     }
     options:{
          is_anonymous: false
     }
     ...
} 
```

However, it seems that this is not the case. I get the response: "expected an Array of String as options."

How can I send a non-anonymous poll?

**Comment by windkh** [comment, marked as answer]
I just copied this from the readme

msg.payload.type = 'poll';
msg.payload.content = "What do you think?";
msg.payload.options = ["A", "B", "C" ];
// you can add optional parameters see https://core.telegram.org/bots/api#sendpoll
msg.payload.optional = { allows_multiple_answers : true };
return msg;
